modules/gdscript/gdscript_byte_codegen.h

```diff
@@ -76,7 +76,37 @@ class GDScriptByteCodeGenerator : public GDScriptCodeGenerator {
 	List<int> temp_stack;
 #endif
 
-	HashMap<Variant, int, VariantHasher, VariantComparator> constant_map;
+	// The default hash_compare is not suitable for storing consts in a HashMap.
+	struct ConstVariantComparator {
```

Comment:
@vnen Should this rather be in [core/variant/variant.h](https://github.com/godotengine/godot/blob/30f2a6d611b1c9a3decae8964c5737e63e63ebce/core/variant/variant.h#L803-L809) instead of being in modules/gdscript/gdscript_byte_codegen.h?

Reply:
No, this is specifically for the bytecode generator, it should stay local.